### PR TITLE
DIG-40772 Adding Beer Case Sizes

### PR DESCRIPTION
**Pull request by tylerbrownhenry**
# [DIG-40772](https://totalwine.atlassian.net/browse/DIG-40772)

## Changes.
* Adding 7 beer case icons sizes as options 
	"12ca"
	"15ca"
	 "18ca"
	"24ca"
	"28ca"
	"30ca"
	"36ca"
* Renaming Keg Options to easily translate from bloom reach parametes '1/2 Keg', '1/4 Keg' and '1/6 Keg'
	"12keg"
	"14keg"
         "16keg"

## Screen Shots
Screen Shot 2020-09-01 at 5 09 07 PM
Screen Shot 2020-09-01 at 5 09 12 PM
Screen Shot 2020-09-01 at 5 09 28 PM
Screen Shot 2020-09-01 at 5 09 22 PM
Screen Shot 2020-09-01 at 5 09 19 PM
Screen Shot 2020-09-01 at 5 09 10 PM
Screen Shot 2020-09-01 at 5 13 54 PM

### CODE REVIEW

File: data.go [hunk 1]
```diff
@@ -24,45 +24,52 @@ import (
...
+	optIndicatorSize300ml        = "300ml"
+	optIndicatorSize375ml        = "375ml"
+	optIndicatorSize500ml        = "500ml"
+	optIndicatorSize2pack        = "2pack"
+	optIndicatorSize3pack        = "3pack"
+	optIndicatorSize4pack        = "4pack"
+	optIndicatorSize6pack        = "6pack"
+	optIndicatorSize8pack        = "8pack"
+	optIndicatorSize9pack        = "9pack"
+	optIndicatorSize10pack       = "10pack"
```
Comment (kmcrawford) [inline data.go]: why was this changed from pk to pack?

Comment (tylerbrownhenry) [reply data.go]: updated the option parameters

**Review by kmcrawford — APPROVED** [review]
